File: content/api-documentation/how-to/orders/examples/getOrders.go
package main

import (
	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
)

func init() {
	alpaca.SetBaseUrl("https://paper-api.alpaca.markets")
}

func main() {
	// Get the last 100 of our closed orders
	status := "closed"
	limit := 100
	closed_orders, err := alpaca.ListOrders(&status, nil, &limit)
	if err != nil {
		panic(err)
	}

	// Get only the closed orders for a particular stock
	var aaplOrders []alpaca.Order
	for _, order := range closed_orders {
		if order.Symbol == "AAPL" {
			aaplOrders = append(aaplOrders, order)
		}
	}
}

File: content/api-documentation/how-to/orders/examples/getOrders.js
const Alpaca = require('@alpacahq/alpaca-trade-api')
const alpaca = new Alpaca()

// Get the last 100 of our closed orders
const closedOrders = alpaca.getOrders({
    status: 'closed',
    limit: 100,
    nested: true  // show nested multi-leg orders
}).then((closedOrders) => {
    // Get only the closed orders for a particular stock
    const closedAaplOrders = closedOrders.filter(order => order.symbol == 'AAPL')
    console.log(closedAaplOrders)
})

